cmake/BuildLibCURL.cmake

```diff
@@ -166,6 +171,7 @@ macro(curl_build)
     list(APPEND LIBCURL_CMAKE_FLAGS "-DCURL_DISABLE_MQTT=ON")
     list(APPEND LIBCURL_CMAKE_FLAGS "-DCURL_DISABLE_SMTP=OFF")
     list(APPEND LIBCURL_CMAKE_FLAGS "-DCURL_DISABLE_ALTSVC=ON")
+    list(APPEND LIBCURL_CMAKE_FLAGS "-DCURL_DISABLE_SRP=OFF")
```

Comment:
> CURL_DISABLE_SRP - Disables cookies support.

It is not about cookies, it is about [TLS-SPR](https://en.wikipedia.org/wiki/TLS-SRP) (a password based auth on the TLS level).

Reply:
Fixed commit message, missed this incorrect information.